src/python/pants/notes/master.rst

```diff
@@ -5,6 +5,84 @@ This document describes development releases which occur weekly from master, and
 not undergone the vetting associated with ``stable`` releases.
 
 
+1.15.0.dev4 (3/15/2019)
+-----------------------
+
```

Comment:
Should we add a brief note here about this being the first release that supports Python 3? I really like in these release notes how we provide a summary: https://www.pantsbuild.org/notes-1.3.x.html.

Maybe pull some language from the message I sent in #python3 over Slack.

Reply:
We should with the final release I think.

Reply:
That's fine with me for waiting, especially for the longer message about how to consume Python 3.

I still think one sentence like this would be nice: `Note this is the first version that (experimentally) supports running Pants with Python 3.6 or Python 3.7`. Provides a bit more context to folks when reading the release notes for 1.15.0.x that they can't use Py3 with 1.15.0.dev3, for example, even though the final cut allows it.

Either way, looks good!

Reply:
So, absent the `setup` changes, someone reading this would just wonder: "how do I do that?".

I'm happy to put that blurb in the release email, which you can then reply to with the instructions for use?

Reply:
Sure, sounds good!